### PR TITLE
chore: Create a txt file for pure components in react

### PR DESCRIPTION
**Pull request by AlexandreResende**

### CODE REVIEW

**Comment by AlexandreResende** [comment]
This is about #14 and #7

File: document/pureComponent.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// This is a simple declaration of a Pure Component in React
```
Comment (Swap76) [inline document/pureComponent.txt]: Good Addition

File: document/component.txt [hunk 2]
```diff
@@ -1,26 +1,26 @@
-We divide the content on a page into diffrent smaller components for easy of use and maintanibility
+We divide the content on a page into different smaller components for easy of use and maintainability
```
Comment (Swap76) [inline document/component.txt]: Overall Good amount of work done

**Comment by Swap76** [comment]
@AlexandreResende  It was great working with you.

Looking forward to working with you :-)

Star to the repo is highly appreciated.

**Comment by Swap76** [comment]
@AlexandreResende The work you have done for the project is highly appreciated. Thanks :-)

Looking forward to working with you on my new project related to Deno https://github.com/Swap76/Learn-Deno/ and together making this better.

Star to the repo is highly appreciated.

**Pull request merged**